chcemvediet/apps/inforequests/tests/test_views/test_detail.py

```diff
@@ -112,20 +121,12 @@ def test_inforequest_with_undecided_email(self):
         self._create_inforequest_email(inforequest=inforequest)
 
         self._login_user()
-        response = self.client.get(reverse(u'inforequests:detail', args=(inforequest.pk,)))
+        response = self.client.get(reverse(u'inforequests:detail', args=(inforequest.slug, inforequest.pk,)))
 
         # User can decide
-        self.assertTemplateUsed(response, u'inforequests/detail_decide.html')
-        self.assertTemplateUsed(response, u'inforequests/detail_undecided.html')
+        self.assertTemplateUsed(response, u'inforequests/detail/texts/add_email.html')
         # User may not act
-        self.assertTemplateNotUsed(response, u'inforequests/detail_add_smail.html')
-        self.assertTemplateNotUsed(response, u'inforequests/detail_new_action.html')
-
-        templates = defaultdict(list)
-        for template in response.templates:
-            templates[template.name].append(template)
-
-        self.assertTemplateUsedCount(response, u'inforequests/detail_email.html', 3)
+        self.assertTemplateNotUsed(response, u'inforequests/detail/texts/add_smail.html')
```

Comment:
@viliambalaz V casti "may not act" by sa zislo otestovat, ze sa nezobrazia operacie z `templates/inforequests/detail/action/controls.html`. Resp v "may act" v `test_inforequest_without_undecided_email` ze sa zobrazia.

Analogicky v dalsich testoch, kde sa testuje, ze user (ne)ma vidiet moznost robit nejake operacie nad ziadostou.

Reply:
:ok:, pridal som samostatne testy pre toto overenie. Testy `test_closed_inforequest_with_undecided_email_applicant_can_*` som nenapisal, prisli mi zbytocne. pretoze v temple `/detail/action/controls.html` je podmienka `inforequest.closed and not inforequest.has_undecided_emails`. (**and**)